### DISCUSSION TITLE
RabbitMQ Operator behavior during external API manipulation

### DISCUSSION THREAD

**Discussion by dima-claroty** [rabbitmq/cluster-operator#1656] [Q&A] [answered]
Hello,

We wanted to start using RabbitMQ operator for our production workload.
However, we manipulate all internal arch (queues, exchanges, vhosts) via API calls from application side.

Is there any danger (now or in the future) of the operator overriding these configurations?
We will not use any of the CRDs besides the **rabbitmqclusters.rabbitmq.com** one, we will however modify it for scaling reasons.

Kind regards & thanks in advance 🙏

**Comment by mkuratczyk** [comment, marked as answer]
I'm not sure I understand the question but I don't see how the cluster operator could interfere in any way. It just deploys a RabbitMQ cluster. You are free to use the API as always.

I'd appreciate if you could share what you are planning to modify and why.

**Reply by dima-claroty** [reply]
Appreciate the quick response,

We will continue using the rabbitmqclusters crd in the documented way, we'll modify the manifest not the operator.
